pages/about.html

```diff
@@ -0,0 +1,140 @@
+                    </div>
+                </div>
+                <div class="headline container">
+                    <p class="title-sub">"Hello! Developers"</p>
+                    <h1 class="font-cocogoose title">CODE 4 EASY DEV BOOTCAMP 2022</h1>
+                    <p class="caption">
+                        Code 4 Easy Bootcamp takes all its students from knowing nothing about
+                        programming or web programming or anything related to independent
+                        software developer within 5 months of intensive learning and coding.
+                    </p>
+                    <p class="contact-info">Please contact us per Email for any further questions about Code 4 Easy
+                        Bootcamp</p>
+                    <a href="email:ishmaelkargbo5@outlook.com">Ishmaelkargbo5@gmail.com</a>
+                </div>
+            </div>
+        </header>
+        <section class="past logo-section">
+            <div class="container">
+                <div class="title-div">
+                    <p class="title">Code 4 Easy DevBootcamp Logo</p>
+                    <hr class="red-line">
+                </div>
+                <div>
+                    <p class="des">
+                        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quod quae molestiae aliquid sint. Culpa error dolorem amet. Voluptatum, dolore nostrum impedit accusantium ex dolorum quo distinctio aliquid, natus sunt ducimus!
+                        Veritatis neque temporibus sunt molestiae assumenda est quos sit id voluptate quam accusamus consequuntur, reprehenderit, quibusdam suscipit labore quisquam perspiciatis asperiores.
+                    </p>
+                    <div class="logo-des">
+                        <p class="logo">Code 4 Easy</p>
+                        <p class="time">9th - 13th May 2022 | Freetown, Sierra Leone</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="past container">
+            <div class="title-div">
+                <p class="title">See the past Code 4 Easy Bootcamp</p>
+                <hr class="red-line">
+            </div>
+            <ul class="list">
+                <li>
+                    <div class="card">
+                        <img src="../assets/images/about1.jpg" alt="about1">
+                        <div class="detail">
+                            <h3 class="font-cocogoose">2020</h3>
+                            <p>Code 4 Easy Bootcamp in Bo</p>
+                        </div>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="../assets/images/about2.jpg" alt="about2">
+                        <div class="detail">
+                            <h3 class="font-cocogoose">2019</h3>
+                            <p>Code 4 Easy Bootcamp in Kenema</p>
+                        </div>
```

Comment:
- [ ] OPTIONAL: Please center the text in this card:
<img width="367" alt="Screenshot 2022-04-15 at 12 36 37" src="https://user-images.githubusercontent.com/77038610/163547499-4a76a57a-7792-4195-8a63-ad7031624c16.png">

- [ ] OPTIONAL: I can see that partners section was deleted from both, mobile and desktop sizes, but based on requirements you had to delete it only for desktop sizes: 
 
Your design  | Behance design
------------- | -------------
<img width="379" alt="Screenshot 2022-04-15 at 12 38 05" src="https://user-images.githubusercontent.com/77038610/163547666-4147c178-f657-49ca-a6d1-cf18738a831e.png">  | <img width="453" alt="Screenshot 2022-04-15 at 12 38 29" src="https://user-images.githubusercontent.com/77038610/163547715-3e381dcb-fd21-4a3c-b635-ce0c49cee173.png">

Reply:
thanks 😊